portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -278,14 +278,29 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be previewed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test ParagraphFieldCanBePreviewed {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133782 ParagraphFieldCanBePreviewed pending implementation
 
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
+
+		FormsAdminNavigator.gotoPreviewForm();
+
+		FormFields.viewParagraphBodyContent(key_bodyName = "Test");
```

Comment:
@piresdm, please rename:

- `key_bodyName` to `bodyName`

Reply:
@natocesarrego done!